lstchain/scripts/lstchain_add_source_dependent_parameters.py

```diff
@@ -1,4 +1,16 @@
-# script to add the source dependent parameters to a DL1 file
+#!/usr/bin/env python3
+
+"""
+Script to add the source dependent parameters to a DL1 file
+
+- Input: DL1 data file.
+- Output: DL1 data file.
```

Comment:
technically no output right? It mutates the input

Reply:
right, I changed the description